modules/userId.js

```diff
@@ -75,6 +75,7 @@ import {getGlobal} from '../src/prebidGlobal.js';
 import {gdprDataHandler} from '../src/adapterManager.js';
 import {unifiedIdSubmodule} from './unifiedIdSystem.js';
 import {pubCommonIdSubmodule} from './pubCommonIdSystem.js';
+import {id5IdSubmodule} from './id5IdSystem.js';
```

Comment:
Sorry one small change, the submodule should be initialized in `id5System.js`. You can see an example of how digitrust implemented it: https://github.com/prebid/Prebid.js/pull/3867/files#diff-9d61416118df40ac8ed0866aaaa0b867R307

Reply:
Sure things, this is done. I just respected the comment at the end of `userId.js`, and did like `unifiedIdSystem.js` and `pubCommonIdSystem.js`. It seems to me that we should move those initialisations too and remove the comment, isn't it?